### PR TITLE
Reduced the fontSize for the small devices

### PR DESCRIPTION
**Pull request by the-shivam-gupta**
To optimise for smaller devices, the font size for the copyright section has been reduced.

## Fixes Issue
#1625

## Screenshots
image

### CODE REVIEW

File: components/Footer/Footer.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export const Footer: FC = () => {
   } `
 
   return (
-    <footer className="z-10 mb-4 flex w-full items-baseline justify-center rounded-lg py-2 backdrop-blur-md sm:py-3">
+    <footer className="z-10 mb-4 flex w-full items-baseline justify-center rounded-lg py-2 backdrop-blur-md sm:py-3 text-sm md:text-base lg:text-lg xl:text-xl">
```
Comment (rupali-codes) [inline components/Footer/Footer.tsx:14]:
```suggestion
    <footer className="z-10 mb-4 flex w-full items-baseline justify-center rounded-lg py-2 backdrop-blur-md sm:py-3 text-sm">
```

**Review by CBID2 — APPROVED** [review]
Hi @the-shivam-gupta! I applied @rupali-codes' suggestions. Things look pretty good! :) LGTM! :)

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
I think it would be better to align this in center. It would be helpful for everyone and make sure to give it some padding from both sides.

image

**Comment by the-shivam-gupta** [comment]
@Anmol-Baranwal, I have made the changes please review it.

**Review by Anmol-Baranwal — APPROVED** [review]
🚀 Looks good to me.

> Thanks for the changes.

**Comment by the-shivam-gupta** [comment]
@rupali-codes, review it

**Review by CBID2 — APPROVED** [review]
LGTM

**Comment by CBID2** [comment]
Thanks for contributing @the-shivam-gupta! If you stay updated on LinksHub's progress, join our [Discord](https://discord.com/invite/NvK67YnJX5)! :) We loved to see more of your ideas! 😊

**Pull request merged**